### DISCUSSION TITLE
std::filesystem not recognized as a namespace

### DISCUSSION THREAD

**Discussion by Balrrach** [clangd/clangd#1069] [Q&A]
Using `namespace fs = std::filesystem;` in a c++ file gives the following error: `1. Expected namespace name`. Project compiles without trouble, why is this happening?, and, how can I fix it?.

**Comment by HighCommander4** [comment]
Please provide a complete example file, and [clangd logs](https://clangd.llvm.org/troubleshooting#gathering-logs) to allow further investigation.
